src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -38,8 +38,8 @@ PROGRAM_INFO("L2-regularized Logistic Regression and Prediction",
     "those things at once.  In addition, this program allows classification on "
     "a test dataset (specified with the " + PRINT_PARAM_STRING("test") + " "
     "parameter) and the classification results may be saved with the " +
-    PRINT_PARAM_STRING("output") + " output parameter.  The trained logistic "
-    "regression model may be saved using the " +
+    PRINT_PARAM_STRING("predictions") + " predictions parameter."
```

Comment:
Actually we should leave this as `output parameter` and not change to `predictions parameter`.  Here we are referring to the fact that the parameter `predictions` is an output parameter (as opposed to an input parameter).